src/modules/Routing.rb

```diff
@@ -205,6 +205,35 @@ def WriteIPForwarding
       write_ipv6_forwarding(@Forward_v6)
     end
 
+    # Converts routes config as read from system into well-defined format
```

Comment:
apart of that ```normalize_routes``` works on top of an array of routes, the basic difference is that ```normalize_routes``` should turn route definition into form which is expected in ```convert_route_conf``` - that "destination" field carries only network ip and netmask or prefix is placed in "destination"